### PR TITLE
duo documentation, cleanup mono documentation

### PR DESCRIPTION
**Pull request by ronakice**

### CODE REVIEW

File: docs/experiments-duot5-tpu.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+
+The files are made available in our [bucket](https://console.cloud.google.com/storage/browser/castorini/duot5/data).
+
+Note that there might be a memory issue if the duoT5 input file is too large for the memory in the instance. We thus split the input file into multiple files.
+
+```
+split --suffix-length 3 --numeric-suffixes --lines 500000 ${DATA_DIR}/query_docs_triples.dev.small.txt ${DATA_DIR}/query_docs_triples.dev.small.txt
+```
+
+For `query_docs_triples.dev.small.txt`, we will get 9 files after split. i.e. (`query_docs_triples.dev.small.txt000` to `query_docs_triples.dev.small.txt034`).
```
Comment (rodrigonogueira4) [inline docs/experiments-duot5-tpu.md:104]: We say 9 files here but below we iterate in 35 files ({000..034}. Could please also confirm if `--lines 500000` is correct?

File: pygaggle/data/convert_duot5_output_to_msmarco_run.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import collections
+import numpy as np
+from tqdm import tqdm
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--t5_output", type=str, required=True,
+                    help="tsv file with two columns, <label> and <score>")
+parser.add_argument("--t5_output_ids", type=str, required=True,
+                    help="tsv file with five columns <query_id>, <doc_id_a>, <doc_id_b>, <rank_a> and <rank_b>")
+parser.add_argument("--mono_run", type=str, required=True,
```
Comment (rodrigonogueira4) [inline pygaggle/data/convert_duot5_output_to_msmarco_run.py:14]: `input_run`  as it does not necessarily come from monoT5?

File: docs/experiments-duot5-tpu.md (hunk 1 above)
Comment (ronakice) [reply docs/experiments-duot5-tpu.md:104]: Oops should say 35 files, will fix it!

File: pygaggle/data/convert_duot5_output_to_msmarco_run.py (hunk 2 above)
Comment (ronakice) [reply pygaggle/data/convert_duot5_output_to_msmarco_run.py:14]: true, will fix this too

**Review by rodrigonogueira4 — APPROVED** [review]
Awesome! thanks a lot for adding this documentation+code.

**Pull request merged**